src/activities/views.clj

```diff
@@ -215,23 +203,19 @@
 ;;    [:ul
 ;;     (map (fn [a-uuid]))]])
 
+;; TODO authorization
 (defn edit-activity-form
-  [req]
-  (let [activity    (activity/req->activity req)
-        activity-id (:crux.db/id activity)
-        title       (:activity/title activity)
-        description (:activity/description activity)
-        inst        (:activity/date-time activity)
-        datetime    (-> inst
-                        (time/local-date-time (time/zone-id "UTC"))
-                        (time/truncate-to :minutes)
-                        (time/format))
-        duration    (:activity/duration activity)
-        capacity    (:activity/capacity activity)]
+  [{:keys [:crux.db/id ::a/title ::a/description ::a/date-time ::a/duration
+           ::a/capacity]} _ router]
```

Comment:
don't use `_` unless it's an anonymous function. I would, in this case, use `_user-id` so you're still signaling what the field is and that is currently unused.